src/components/tabs/tabs.test.js

```diff
@@ -0,0 +1,603 @@
+        .parent()
+        .should("have.css", "flex-direction", flex)
+        .and("have.css", "height", `${height}px`);
+    });
+
+    it.each([
+      ["default", "row", 40, 67],
+      ["large", "column", 48, 88],
+    ])(
+      "should verify Tabs height and width when size is %s",
+      (size, flex, height, width) => {
+        CypressMountWithProviders(<TabsComponent size={size} />);
+
+        tabById(1)
+          .should("have.css", "height", `${height}px`)
+          .invoke("css", "width")
+          .then(parseFloat)
+          .then(($el) => {
+            expect($el).to.be.gte(width);
+            expect($el).to.be.lt(width + 1);
+          });
+      }
+    );
+
+    it.each([
+      ["extended", true, 1342],
+      ["trimmed", false, 337],
+    ])(
+      "should verify Tabs dividing line is %s when extendedLine prop is %s",
+      (state, bool, width) => {
+        CypressMountWithProviders(<TabsComponent extendedLine={bool} />);
+
+        tabById(1)
+          .parent()
+          .invoke("css", "width")
+          .then(parseFloat)
+          .then(($el) => {
+            expect($el).to.be.gte(width);
+            expect($el).to.be.lt(width + 1);
+          });
+      }
+    );
+
+    it.each([
+      ["off", "none", "none", "none", "none"],
+      ["on", "solid", "none", "solid", "solid"],
+      ["no sides", "solid", "none", "none", "none"],
+    ])(
+      "should verify Tabs borders are set to %s when positioned on top",
+      (state, top, bottom, right, left) => {
+        CypressMountWithProviders(<TabsComponent borders={state} />);
+
+        tabById(1)
+          .children()
+          .should("have.css", "border-top-style", top)
+          .and("have.css", "border-bottom-style", bottom)
+          .and("have.css", "border-right-style", right)
+          .and("have.css", "border-left-style", left);
+        tabById(1).parent().should("have.css", "flex-direction", "row");
+      }
+    );
+
+    it.each([
+      ["off", "none", "none", "none", "none"],
+      ["on", "solid", "solid", "none", "solid"],
+      ["no sides", "solid", "solid", "none", "none"],
+    ])(
+      "should verify Tabs borders are set to %s when positioned on left",
+      (state, top, bottom, right, left) => {
+        CypressMountWithProviders(
+          <TabsComponent position="left" borders={state} />
+        );
+
+        tabById(1)
+          .children()
+          .should("have.css", "border-top-style", top)
+          .and("have.css", "border-bottom-style", bottom)
+          .and("have.css", "border-right-style", right)
+          .and("have.css", "border-left-style", left);
+        tabById(1).parent().should("have.css", "flex-direction", "column");
+      }
+    );
+
+    it("should verify Tabs width is set by headerWidth prop", () => {
+      CypressMountWithProviders(
+        <TabsComponent headerWidth="440px" align="left" position="left" />
+      );
+
+      tabById(1).children().should("have.css", "width", "400px");
```

Comment:
is the `400px` a correct value? because we have set `440px` for `headerWidth`

Reply:
I was checking the wrong width, fixed now.